src/data/markdown/docs/03 cloud/09 Optional Add-ons/01 Private load zones.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Private load zones'
+excerpt: 'How to set up private load zones in k6 Cloud'
+---
+
+A private load zone is a special location that we set up for you on k6 Cloud. This location corresponds to availability zones you've selected on your own AWS account. With private load zones, you can start tests on k6 Cloud but have them execute on _your own cloud infrastructure_, provisioned within AWS.
```

Comment:
We need to make it very clear that this is an Enterprise feature, including a link to our Pricing page so they can be aware it starts at $25k.

Reply:
You can reuse this UI pattern:
<img width="892" alt="Screenshot 2022-04-20 at 12 56 25" src="https://user-images.githubusercontent.com/825430/164216160-cc940376-779c-4ff8-bc1a-04d2dd9a8ff9.png">

Reply:
Done in the latest commit!